### ISSUE TITLE
Connector Problem (Job Runner) with Multiple Odoo Instances

### ISSUE THREAD

**Issue by aignermi** [OCA/connector-magento#233]
We just stumbled across a problem with job runner, when we created a second odoo database on one of our instances (second db on one setup). Jobs were just not picked up by the job runner. They just stayed in the Pending status. Is the job runner even capable of working for multiple installations?

**Comment by guewen** [comment]
What do you have in the `--database` or `db_name` option?

**Comment by sbidoul** [comment]
> Is the job runner even capable of working for multiple installations

Yes it is. But you need to restart odoo after creating new databases and installing connector on them, as the runner does not actively look for new databases after startup.

https://github.com/OCA/connector/blob/f67cb0729f1defef1e0ec329b17e0e733247f806/connector/jobrunner/runner.py#L350

**Comment by aignermi** [comment]
@guewen for the db_name we have the following: db_name = False

and @sbidoul i restarted odoo a few times but it did not pick it upt!
